src/module/helperCode/Equipment Checker.py

```
@@ -42,7 +42,8 @@ def magicPrice (price,magicBonus):
 
                         # Applies the default magic bonus if in the name
                         data["system"]["armor"]["magicalBonus"] = magicBonus
-
+                        data["system"]["unidentified"]["description"] = "<p><em>This armor is unusually strong and well-crafted. It is most likely magic, though you can’t identify the specific enchantment.</em></p>"
```

Ruff (0.7.0)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)